Doc/whatsnew/3.14.rst

```diff
@@ -210,6 +210,10 @@ configuration mechanisms).
 
 Other language changes
 ======================
+* Constant comparsion expressions are now folded and evaluated before runtime.
```

Comment:
Do we have a terminology that would be like "at compilation time" instead of "before runtime"? (using "compilation" time may cause users to think that it's only when `.pyc` files are created but this can be skipped with `-B` I think?)